docs/index/index.rst

```diff
@@ -25,7 +25,7 @@ The GeoTools user guide is available online:
     The user guide is available for `download <http://sourceforge.net/projects/geotools/files/>`_
     as a zip file.
        
-`Tutorials <http://docs.geotools.org/latest/userguide/tutorial/>`_ ( `stable <http://docs.geotools.org/stable/tutorials/>`_ | `maintenance <http://docs.geotools.org/maintenance/tutorials/>`_ )
+`Tutorials <http://docs.geotools.org/latest/userguide/tutorial/>`_ ( `stable <http://docs.geotools.org/stable/userguide/tutorials/>`_ | `maintenance <http://docs.geotools.org/maintenance/userguide/tutorials/>`_ )
```

Comment:
I think it needs to be tutorial (singular) for both `stable` and `maintenance`, as for the main link